jekyll/_includes/global-nav.html

```diff
@@ -44,6 +44,12 @@
             <div class="lang-dropdown">
               <a id="globalNavLangEng" class="lang-option">English</a>
               <a id="globalNavLangJap" class="lang-option">日本語</a>
+              <div class="lang-request-form">
+                <span class="lang-form-title">Request a language</span>
+                <input id="lang-req" class="lang-form" type="text" placeholder="Language" maxLength='12'
+                  autocomplete="off">
+                <a id="submit-btn">Submit</a>
```

Comment:
I am unsure why this is a a tag but no href associated to it. If its a form submit can we use something like the following
```
<form id="myForm" action="/action_page.php">
  First name: <input type="text" name="fname"><br>
  Last name: <input type="text" name="lname"><br><br>
  <input type="button" onclick="myFunction()" value="Submit form">
</form>

<script>
function myFunction() {
  document.getElementById("myForm").submit();
}
</script>
```

Reply:
`submit-btn` is just used to send the input for the language request to Amplitude on click. I used an a tag because I thought it would be the easiest to style as `<input type="button"> ` and `<button>` both come with some inherent styles I believe. It would definitely be more semantic to use the other two. 

We arent actually submitting a form  with a first and last name as `<input>` is just being used in this case for users to type in a language.

Reply:
Can you use a <div> instead of a <a> ?

Reply:
since its a clickable element, I think using an a tag here would be a better choice as it conveys clickable semantics and is also much easier to style than using a button element. I think sites like twitter also uses a tags for some of there buttons.